### PR TITLE
LWDEV-7718: Added settings modal window

### PR DESCRIPTION
**Pull request by PaulRumyantsevItr**
- added settings modal component
- splitted old settings by separate components
- added components for Tabs
- added and customized Dropdown component from lykke-react-components repository
- added Enzyme library
- covered new components by tests

### CODE REVIEW

File: src/components/CustomDropdown/styles.ts [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+    > div {
+      border: none !important;
+    }
+
+    li.dropdown-list__item {
+      border: 1px solid ${colors.greyBorder};
+      border-bottom: 0;
+      line-height: normal;
+      color: #f5f6f7;
```
Comment (sergeyschegolev) [inline src/components/CustomDropdown/styles.ts]: pls, use constant from styled.ts(colors)

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -73,17 +72,17 @@ const Header: React.SFC<HeaderProps> = ({
...
-                  </span>
-                  {settingsStore.showSettings ? <SettingsModal /> : null}
-                </HeaderItem>
-              </ClickOutside>
+              <HeaderItem>
+                <span
+                  className="hidden-xs settings"
+                  onClick={handleToggleSettings}
+                >
+                  <Icon name={'cogwheel'} color={'#8c94a0'} />
```
Comment (sergeyschegolev) [inline src/components/Header/Header.tsx]: the same here

File: src/components/Settings/styles.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+import {Button, Modal, ModalContent} from '../Modal/styles';
+import styled from '../styled';
+// tslint:disable-next-line:no-var-requires
+const {Flex} = require('grid-styled');
+
+export const StyledSettingsModal = styled(Modal)`
+  min-width: ${rem(730)};
+  min-height: ${rem(150)};
+  padding: ${rem(28)} ${rem(28)};
+  background-color: #3e3e3e;
```
Comment (sergeyschegolev) [inline src/components/Settings/styles.ts]: the same

File: src/components/Settings/styles.ts [hunk 4]
```diff
@@ -33,6 +33,18 @@ export const SettingsListItemTitle = styled.div`
...
+
+  &.left-align > div {
+    width: auto;
+
+    &:first-child > div {
+      padding-left: 0;
+    }
+
+    > div {
+      padding: 0 30px;
```
Comment (Georg3MK) [inline src/components/Settings/styles.ts]: Maybe we should use ${rem(30)} here

**Pull request merged**